### PR TITLE
Soracom add line deprecation message

### PR DESCRIPTION
**Pull request by shogo-soracom**
Continuation of https://github.com/soracom/grafana/pull/26 which was accidentally merged...

Reflected the recommended changes

Screenshot from 2024-11-05 09-55-40

### CODE REVIEW

**Comment by shogo-soracom** [comment]
[sc-126358]

**Review by tau6283 — APPROVED** [review]
LGTM thanks!

File: pkg/services/ngalert/notifier/channels_config/plugin.go [hunk 1]
```diff
@@ -7,6 +7,7 @@ type NotifierPlugin struct {
 	Heading     string           `json:"heading"`
 	Description string           `json:"description"`
 	Info        string           `json:"info"`
+	Alert       []string         `json:"alert"`
```
Comment (cinkster) [inline pkg/services/ngalert/notifier/channels_config/plugin.go:10]: Im assuming this struct is only ever used for frontend presentation?  (not saved in db or unmarshalled from json)

**Review by cinkster — APPROVED** [review]
LGTM just comments

File: pkg/services/ngalert/notifier/channels_config/plugin.go (hunk 1 above)
Comment (shogo-soracom) [reply pkg/services/ngalert/notifier/channels_config/plugin.go:10]: The ngalert NotifierPlugin only appears in channels_config.GetAvailableNotifiers, which is just used to send the list of options to the front-end

File: pkg/services/ngalert/notifier/channels_config/available_channels.go [hunk 2]
```diff
@@ -948,6 +948,10 @@ func GetAvailableNotifiers() []*NotifierPlugin {
 			Name:        "LINE",
 			Description: "Send notifications to LINE notify",
 			Heading:     "LINE notify settings",
+			Alert: []string{
+				"LINE Notify API の提供終了に伴い、2025 年 3 月 31 日以降は LINE 通知を利用できません。別の Contact Point の利用を検討してください。",
+				"After the LINE Notify API reaches EOL on March 31, 2025, LINE alerting will no longer function. Please consider using another contact point.",
```
[the remark this answers is not in the document]
Comment (shogo-soracom) [reply pkg/services/ngalert/notifier/channels_config/available_channels.go:953]: This is a good point I was forgetting. Should I point towards our documentation? Or should I just point straight to https://notify-bot.line.me/closing-announce ?

I would also like @tau6283 's opinion on this one I think

**Comment by shogo-soracom** [comment]
image

**Comment by shogo-soracom** [comment]
image

**Pull request merged**